http/server.go

```diff
@@ -529,6 +530,25 @@ func (h *DrandHandler) Health(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(b)
 }
 
+func (h *DrandHandler) ChainHashes(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could we have a quick test to make sure it's working as expected ?

Reply:
Yes, we can. I wanted to know if the idea was correct before creating new tests for this.

Reply:
Added!